starknet-core/src/types/typed_data/mod.rs

```diff
@@ -491,6 +491,59 @@ mod tests {
         );
     }
 
+    #[cfg(test)]
+    mod message_hash {
```

Comment:
There seems to be no reason to nest a module? Also we're already in a `#[cfg(test)]` context here.

Reply:
Yes, it just a way to structure tests by modules. But I can change that